patches/chrome-installer-mini_installer-mini_installer.cc.patch

```diff
@@ -11,25 +11,28 @@ index 73e6240800f3c72bd96c91971c53a19516e1b0f1..e0049fab53dcb2bb8fe237523bf6eb98
  // Opens the Google Update ClientState key. If |binaries| is false, opens the
  // key for Google Chrome or Chrome SxS (canary). If |binaries| is true and an
  // existing multi-install Chrome is being updated, opens the key for the
-@@ -144,6 +144,138 @@ void SetInstallerFlags(const Configuration& configuration) {
+@@ -144,6 +144,146 @@ void SetInstallerFlags(const Configuration& configuration) {
  }
  #endif  // BUILDFLAG(GOOGLE_CHROME_BRANDING)
  
 +#if defined(BRAVE_CHROMIUM_BUILD)
++#define StandardReferralCodeLen 6
```

Comment:
As you touch this file, let's cleanup this patch file together. :)
We can delete whole this file by overriding this file.

Reply:
I wrote incorrectly - this patch can be one line.

Reply:
FYI - patching to `BUILDFLAG` also could be removed by adding this in overriding file.
```
#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
#undef BUILDFLAG_INTERNAL_GOOGLE_CHROME_BRANDING
#define BUILDFLAG_INTERNAL_GOOGLE_CHROME_BRANDING() (1)
#endif
```